iosslsniffer/__main__.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+import click
+import coloredlogs
+from pymobiledevice3.lockdown import create_using_usbmux
+from rpcclient.client_factory import create_tcp_client
+
+from iosslsniffer.sniffer import Filters, Sniffer
+
+GLOBAL_PREFERENCE = '/private/var/Managed Preferences/mobile/.GlobalPreferences.plist'
+CFNETWORK_KEY = 'AppleCFNetworkDiagnosticLogging'
+
+coloredlogs.install(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def default_none(ctx, _, value):
+    if len(value) == 0:
+        return None
+    else:
+        return value
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command('setup')
+@click.option('rpc', '-r', '--rpc', nargs=2, type=click.Tuple([str, int]), default=('127.0.0.1', 5910),
+              help='rpc server ip and port')
+def cli_setup(rpc):
+    """ Setup all prerequisites required inorder to sniff the SSL traffic """
+    rpc = create_tcp_client(rpc[0], rpc[1])
+    global_preferences = rpc.preferences.sc.open(GLOBAL_PREFERENCE)
+    if not global_preferences.get(CFNETWORK_KEY):
+        global_preferences.set(CFNETWORK_KEY, 3)
+        logger.info(f'{CFNETWORK_KEY} is now set, a restart is require')
+        return
+    logger.info(f'{CFNETWORK_KEY} already set')
+    rpc.syslog.set_harlogger_for_all(True)
+
+
+@cli.command('sniff')
+@click.option('out_file', '-o', '--out', default='traffic.pcapng', type=click.STRING,
+              help='outfile name with .pcapng extension')
+@click.option('pids', '-p', '--pid', type=click.INT, multiple=True, callback=default_none, help='filter pid list')
+@click.option('process_names', '-pn', '--process-name', callback=default_none, multiple=True,
+              help='filter process name list')
+@click.option('images', '-i', '--image', multiple=True, callback=default_none, help='filter image list')
+@click.option('--black-list/--white-list', default=True, is_flag=True)
+def cli_sniff(pids, process_names, images, out_file, black_list):
+    """ Sniff the traffic """
+    lockdown = create_using_usbmux()
```

Comment:
import `Command` from `pymobiledevice3` to have it select the device you work on and initialize `lockdown` for you